### PR TITLE
Initial commit

### PR DESCRIPTION
**Pull request by userhas404d**

### CODE REVIEW

File: LICENSE [hunk 1]
```diff
@@ -186,16 +186,16 @@
       same "printed page" as the copyright notice for easier
       identification within third-party archives.
 
-   Copyright [yyyy] [name of copyright owner]
+  Copyright 2019 Maintainers of plus3it/template
```
Comment (lorengordon) [inline LICENSE]: `plus3it/template`? Is the template setting this incorrectly?

File: CHANGELOG.md [hunk 2]
```diff
@@ -0,0 +1,15 @@
+## template Change Log
```
Comment (lorengordon) [inline CHANGELOG.md]: `template` here also seems incorrect...

File: LICENSE (hunk 1 above)
Comment (confusdcodr) [reply LICENSE]: No, we are working through the process on how to do this and don't have a good grasp on how to have the user first create the repo in the org and then fork it with their version

Comment (confusdcodr) [reply LICENSE]: So to address your concern, when the cookiecutter project was run `template` was used as the project name as the terraform-aws-tardigrade-cloudtrail project already existed from a fork being created

File: CHANGELOG.md (hunk 2 above)
Comment (lorengordon) [reply CHANGELOG.md]: roger. how about creating the repo here in the plus3it org strictly from the templater in the first commit, then adding the specific terraform module functionality in the first pr...?

File: LICENSE (hunk 1 above)
Comment (lorengordon) [reply LICENSE]: roger. how about creating the repo here in the plus3it org strictly from the templater in the first commit, then adding the specific terraform module functionality in the first pr...?

Comment (userhas404d) [reply LICENSE]: sounds good, I'll update the working instructions to reflect that.

Comment (confusdcodr) [reply LICENSE]: Could definitely do that. I do still have the task to integrate the terraform-github-repo module with cookiecutter-tfmodule

Comment (lorengordon) [reply LICENSE]: oh, right, i didn't realize the github template wasn't yet integrated. carry on! :D

Comment (confusdcodr) [reply LICENSE]: Yeah, I don't know of an easy way to integrate it

**Pull request merged**